docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -0,0 +1,79 @@
+---
+---
+= Clustered Cache
+ifndef::sourcedir[]
+include::common.adoc[]
+endif::sourcedir[]
+
+ifdef::notBuildingForSite[]
+include::menu.adoc[]
+endif::notBuildingForSite[]
+
+== Introduction
+
+To enable Clustering with Terracotta, firstly you will have to start the http://terracotta.org[Terracotta server] configured with clustered storage.
+And then need a cache manager with clustering capabilities so that the caches can consume the clustered storage.
+
+== Starting Terracotta server
+
+You can either download the Terracotta 5.x distribution from http://terracotta.org/ or use the ehcache-clustered kit generated by the ehcache3 build.
+The example below demonstrates the latter approach.
+
+After building the https://github.com/ehcache/ehcache3[EhCache project], you need to extract the *ehcache-clustered-3.x.y-SNAPSHOT-kit* present in ehcache3/clustered/dist/build/distributions.
+Change to your extracted directory and then execute the *start-tc-server* script:
+
+[source,bash]
+----
+cd terracotta-5.x.y-SNAPSHOT/server/bin
+./start-tc-server.sh
+----
+
+NOTE: You will need to have JAVA_HOME set to JDK8 while starting the Terracotta server.
+
+
+== Creating cache manager with clustering capabilities
+
+For creating the cache manager with clustering support, you will need to provide the clustering service configuration:
+
+[source,java,indent=0]
+----
+include::{sourcedir}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/GettingStarted.java[tag=clusteredCacheManagerExample]
+----
+
+<1> returns the `org.ehcache.config.builders.CacheManagerBuilder` instance;
+<2> Use the `ClusteringServiceConfigurationBuilder`{empty}'s static method `.cluster(URI)` for connecting the cache manager to the clustering storage at the
```

Comment:
Please see if the example can be updated to match the host and port discussed above. If not, when discussing above, document using example.com
Port should be checked - not sure 9540 is correct default.